deployment_scripts/deploy_windows.ps1

```diff
@@ -23,6 +23,10 @@ function Configure-precommit([String] $git_repo_dir)
 		exit
 	}
     Pop-Location
+
+    # Set env variable to skip Swimm verification during pre-commit, Windows not supported yet
+    [System.Environment]::SetEnvironmentVariable('SKIP','swimm-verify',[System.EnvironmentVariableTarget]::User)
```

Comment:
If the user already has a `SKIP` variable configured, this will overwrite it. The `SKIP` environment variable is a comma-separated list of hook ids, so this should append `,swimm-verify` if the `SKIP` variable is already set.